examples/prostate/prostate_3D/README.md

```diff
@@ -0,0 +1,122 @@
+The [submit_job.sh](./submit_job.sh) script follows the pattern:
+```
+bash ./submit_job.sh [config]
+```
+`[config]` is the experiment job that will be submitted for the FL training, in this example, this includes `prostate_central`, `prostate_fedavg`, `prostate_fedprox`, and `prostate_ditto`.  
+
+Note that in order to make it working under most system resource conditions, the current script used regular `Dataset` for data loading in `pt/learners/prostate_learner.py`, which could be slow. If resource permits, it will make the training much faster by replacing it with `CacheDataset`. More information available [here](https://docs.monai.io/en/stable/data.html#cachedataset).  
+
+### 2.3 Centralized training
+To simulate a centralized training baseline, we run FL with 1 client using all the training data. 
+```
+bash start_fl_poc.sh "All"
+bash submit_job.sh prostate_central
+```
+### 2.4 Federated learning
+Start 4 FL clients
+```
+bash start_fl_poc.sh "I2CVB MSD NCI_ISBI_3T NCI_ISBI_Dx"
+```
+#### 2.4.1 FedAvg 
+To run FL with standard [fedAvg](https://arxiv.org/abs/1602.05629), we use
+```
+bash submit_job.sh prostate_fedavg
+```
+#### 2.4.2 FedProx 
+To run FL with [FedProx](https://arxiv.org/abs/1812.06127), which adds a regularizer to the loss used in `SupervisedProstateLearner` (`fedproxloss_mu`), we use
+```
+bash submit_job.sh prostate_fedprox 
+```
+#### 2.4. Ditto 
+To run FL with [Ditto](https://arxiv.org/abs/2012.04221)(official [implementation](https://github.com/litian96/ditto)), which uses a slightly modified version of the prostate Learner implementation, namely the `ProstateDittoLearner`, which decouples local personalized model from global model via an additional model training and a controllable prox term (`ditto_lambda`), we use
+```
+bash submit_job.sh prostate_ditto
+```
+
+## 5. Results on 4 clients for Central vs. FedAvg vs. FedProx vs. Ditto
+In this example, for Central/FedAvg/FedProx, only the global model gets evaluated at each round, and saved as the final model. For Ditto, each client will have its own personalized model, which is validated and saved locally.
+### Validation curve on each site
+
+Let's summarize the result of the experiments run above. We compare the validation scores of 
+the global model for Central/FedAvg/FedProx, and personalized models for Ditto. In this example, each client computes their validation scores using their own
+validation set, and the centralized model computes the validation score using the combined validation set. Please note that due to the limited size of data set, the validation curve can have significant variations across runs.
+
+We provide a script for plotting the tensorboard records, running
+```
+python3 ./result_stat/plot_tensorboard_events.py
+```
+The TensorBoard curves for validation Dice for the 150 epochs (150 rounds, 1 local epochs per round) during training are shown below:
+![All training curve](./figs/all_training.png)
```

Comment:
could add a table with results here as well as in the 2D example.

Reply:
agree, will add

Reply:
added, results different from last version because last time the split is 0.8:0.2:0.0 and current split is 0.5:0.25:0.25